spec/index/index_spec.rb

```diff
@@ -70,6 +70,40 @@
     end
   end
 
+  context '#sort' do
+    let(:idx1) { Daru::Index.new ['speaker', 'mic', 'guitar', 'amp'] }
+    let(:idx2) { Daru::Index.new [:one, 'one', 1, 2, :two] }
```

Comment:
This is over testing. Why do you need so many indexes? Also, ìdx1...4`is a bad naming convention. It does not tell anything about your data.